tools/README.md

```diff
@@ -0,0 +1,41 @@
+# Tools
```

Comment:
The `hack` directory is the conventional place for all things testing, scripts, etc. I think it should go there unless there is a convincing reason otherwise.